### PR TITLE
[wx] Rename 'Insert your YubiKey' to 'Insert YubiKey'

### LINKED ISSUE #1258: Simplify text in inserting Yubikey dialog
Password Safe v1.19-pre on Ubuntu 24.04

1. Manage | Yubikey
2. Bellow dialog is displayed:
image

Suggestion:
- Replace text **Insert your Yubikey** with **Insert Yubikey**

I think we have already discussed this in one of the PRs, but it was some other dialog, not this one. Word "your" can be misleading, why if I don't have "my" Yubikey and I am using the wife's one. Word "your" is redundant.

### PR DESCRIPTION
**Pull request by rafaelx**
Resolves #1258

### CODE REVIEW

File: src/ui/wxWidgets/I18N/pos/pwsafe_ru.po [hunk 1]
```diff
@@ -1251,8 +1251,10 @@ msgstr "Пароль:"
 #: ../../../ui/wxWidgets/SafeCombinationEntryDlg.cpp:211
 #: ../../../ui/wxWidgets/SafeCombinationSetupDlg.cpp:138
 #: ../../../ui/wxWidgets/YubiCfgDlg.cpp:358
+#: ../../../ui/wxWidgets/wxUtilities.cpp:216
+#: ../../../ui/wxWidgets/YubiMixin.cpp:62
 #, fuzzy
-msgid "Insert your YubiKey"
+msgid "Insert YubiKey"
 msgstr "Вставьте ваш YubiKey"
```
Comment (dskrvk) [inline src/ui/wxWidgets/I18N/pos/pwsafe_ru.po:1258]:
```suggestion
msgstr "Вставьте YubiKey"
```
(updated translation)

File: src/ui/wxWidgets/I18N/pos/pwsafe_fr.po [hunk 2]
```diff
@@ -1305,8 +1305,10 @@ msgstr "derniers mots de passe"
 #: ../../../ui/wxWidgets/SafeCombinationEntryDlg.cpp:211
 #: ../../../ui/wxWidgets/SafeCombinationSetupDlg.cpp:138
 #: ../../../ui/wxWidgets/YubiCfgDlg.cpp:358
+#: ../../../ui/wxWidgets/wxUtilities.cpp:216
+#: ../../../ui/wxWidgets/YubiMixin.cpp:62
 #, fuzzy
-msgid "Insert your YubiKey"
+msgid "Insert YubiKey"
 msgstr "Veuillez insérer votre YubiKey"
```
Comment (dskrvk) [inline src/ui/wxWidgets/I18N/pos/pwsafe_fr.po:1312]:
```suggestion
msgstr "Veuillez insérer le YubiKey"
```

**Comment by igor2x** [comment]
@rafaelx, is it really good practice to update .po files in Github?

Workflow, at least like I understand it:
1. Generate .po files
2. Export .po files to Transifex
3. Now you have updated .po files in Github
4. Some translator updates translation in Transifex
5. Translations are imported from Transifex into Github - and changes in step 3 are overridden.

Even in case Russian translations of above manually updated strings are NOT changed in Transifex and one of translators changes some other string and then rony will notice Russian translations were updated in Transifex and imports them into Github and you will also lose above translations.

In my humble opinion translations should never be updating in two sources: Github and Transifex, because then some merging process is required and the merger must know foreign language in order of correctly merge translations.

My suggestions is, update translations in Transifex and remove above commits from PR.

**Comment by ronys** [comment]
I'd rather handle multiple sources, with each translator doing what's convenient, than enforcing Transifex as the only supported translation source.

Practically speaking, the conflicts don't happen, or at least, happen very rarely.

File: src/ui/wxWidgets/I18N/pos/pwsafe_ru.po [hunk 3]
```diff
@@ -5240,8 +5240,11 @@ msgstr ""
 msgid "YubiKey"
 msgstr "YubiKey"
 
+#: src/ui/wxWidgets/wxUtilities.cpp:216 src/ui/wxWidgets/YubiMixin.cpp:62
+msgid "Insert YubiKey"
+msgstr "Вставьте ваш YubiKey"
```
Comment (dskrvk) [inline src/ui/wxWidgets/I18N/pos/pwsafe_ru.po:5245]: One more:
```suggestion
msgstr "Вставьте YubiKey"
```

**Review by dskrvk — APPROVED** [review]

**Pull request merged**